eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/NodeRelayer.scala

```diff
@@ -200,8 +219,10 @@ object NodeRelayer {
    *
    * @param upstream    complete HTLC set received.
    * @param nextPayload relay instructions.
+   * @param paymentId   id of the outgoing payment.
+   * @param settled     true if we already settled the payment upstream.
```

Comment:
I think this name creates confusion with an on-chain settlement. How about:
```suggestion
   * @param fulfilledUpstream     true if we already settled the payment upstream.
```